doc/source/tokudb/tokudb_variables.rst

```diff
@@ -928,10 +928,11 @@ for bulk loader.
   :scope: Global/Session
   :dyn: Yes
   :vartype: Boolean
-  :default: Off
+  :default: OFF
 
-Enables the fast updates feature, which substantially facilitates execution of
-read-modify-write style queries.
+Toggles the fast updates feature ON/OFF for the ``UPDATE`` statement. Fast
```

Comment:
Was there a follow-up DOC task to expand these descriptions?

Reply:
@vlad-lesin plans to make a blog post about fast updates, so it looks reasonable to base updated docs on it.